README.md

```diff
@@ -1,6 +1,38 @@
-Getting started
+# earnings
+
+> Project description
+
+## Table of Contents
+
+1. [Usage](#Usage)
+1. [Requirements](#requirements)
+1. [Development](#development)
+
+## Usage
 
-This uses React through webpack and can be seen on port 3000, so you'll have to
-npm install
 npm run start
-npm run build
\ No newline at end of file
+npm run build
+
+## Requirements
+
+An `nvmrc` file is included if using [nvm](https://github.com/creationix/nvm).
+
+- Node 6.13.0
+- etc
+
+## Development
+
+### Installing Dependencies
+
+From within the root directory:
+
+```sh
+npm install
+```
+## API
+
+| GET               | POST               | PUT                      | DELETE            |
+| ----------------- | ------------------ | ------------------------ | ----------------- |
+| READ              | CREATE             | UPDATE                   | DELETE            |
+| /earnings/:ticker | /earnings/add      | /earnings/update/:ticker | /earnings/:ticker |
```

Comment:
Is there any particular reason POST and PUT are at different endpoints? Could the 4 different verbs be on the same endpoint?

Reply:
@bellcd Thanks, I updated this if you want to review it again before I merge it.